commandapi-platforms/commandapi-bukkit/commandapi-bukkit-vh/src/main/java/dev/jorel/commandapi/CommandAPIVersionHandler.java

```diff
@@ -68,8 +72,23 @@ public interface CommandAPIVersionHandler {
 				case "1.20.3", "1.20.4" -> new NMS_1_20_R3();
 				case "1.20.5", "1.20.6" -> new NMS_1_20_R4();
 				case "1.21", "1.21.1" -> new NMS_1_21_R1();
-				default -> throw new UnsupportedVersionException(version);
+				default -> null;
 			};
+			if (platform != null) {
+				return new LoadContext(platform, () -> {});
+			}
```

Comment:
`LoadContext` could have an additional constructor for the case where nothing extra needs to run:

```java
public record LoadContext(CommandAPIPlatform<?, ?, ?> platform, Runnable context) {
	public LoadContext(CommandAPIPlatform<?, ?, ?> platform) {
		this(platform, () -> {});
	}
}
```

That might make situations like this where no extra logic needs to run clearer.

```suggestion
			if (platform != null) {
				return new LoadContext(platform);
			}
```